src/com/layout/grid/grid.js

```diff
@@ -0,0 +1,22 @@
+import {h, Component} from 'preact/preact';
+import Col from 'com/layout/col/col';
+import Row from 'com/layout/row/row';
+import Container from 'com/layout/container/container';
+
+export default class Grid extends Component {
+  constructor(props) {
```

Comment:
```js
    constructor( props ) {
```

Spaces around arguments are used to differentiate function calls versus definitions.